common.sh

```diff
@@ -35,9 +35,29 @@ delete_pg()
     if [ ${ENABLE_PLACEMENT_GROUP} -eq 0 ] || [ -z $PLACEMENT_GROUP ]; then
         return 0
     fi
-    AWS_DEFAULT_REGION=us-west-2 aws ec2 delete-placement-group \
-        --group-name ${PLACEMENT_GROUP}
-    return $?
+    local ret=0
+    local retry=20
```

Comment:
Add a comment on why 200 seconds is the right length of time here.  I think for PG delete, where instances need to shut down, this is way too short.  I think it's 10 minutes or more if we want to handle metal instances.

Reply:
Thanks, I switched to 10 minutes and added it in the comments.

Reply:
Put the comment up here, where you're setting the retry count.  Also, put a sleep_time variable up here to match.  Try to keep the configuration and code a little separate.

Reply:
Fixed in latest revision.

Reply:
Fixed in latest revision.